tests/e2e/scenarios/test_journey_coverage.py

```
@@ -322,17 +336,61 @@ def test_provider_journey_runs_preserve_isolated_repeat_cases():
+
+
+@pytest.mark.parametrize(
+    ("value", "expected"),
+    (
+        ("reverse", True),
+        ("REVERSE", True),
+        ("  reverse  ", True),
+        ("forward", False),
+        ("", False),
+        (None, False),
+    ),
```

Ruff (0.15.21)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list` of `tuple`

Use `list` of `tuple` for parameter values

(PT007)